src/OpenLoco/Windows/PromptOkCancelWindow.cpp

```diff
@@ -42,7 +31,7 @@ namespace OpenLoco::Ui::Windows::PromptOkCancel
 
     static Widget _widgets[] = {
         makeWidget({ 0, 0 }, { 280, 92 }, WidgetType::panel, WindowColour::primary),
-        makeWidget({ 1, 1 }, { 278, 13 }, WidgetType::caption_22, WindowColour::primary, StringIds::buffer_2039),
+        makeWidget({ 1, 1 }, { 278, 13 }, WidgetType::caption_22, WindowColour::primary),
```

Comment:
You've remove the buffer from the title but kept it in the prepare draw? I think you meant to keep the title.